kaldb/src/main/java/com/slack/kaldb/server/KaldbIndexer.java

```diff
@@ -110,7 +112,10 @@ private long indexerPreStart() throws Exception {
             meterRegistry);
 
     long currentHeadOffsetForPartition = kafkaConsumer.getEndOffSetForPartition();
```

Comment:
I'm not a huge fan of calling these head/tail, especially since we seem to have inverted the common definitions. Tail is generally the most recent messages, head the oldest. I realize you haven't started this pattern, but we are extending this further now.

Maybe using similar terms as Kafka is a good plan here - end / beginning.